msvc/dependencies/zycore/Zycore.vcxproj

```diff
@@ -115,6 +115,9 @@
     <SupportsPackaging>false</SupportsPackaging>
     <SpectreMitigation>false</SpectreMitigation>
     <Driver_SpectreMitigation>false</Driver_SpectreMitigation>
+    <LatestTargetPlatformVersion>$([Microsoft.Build.Utilities.ToolLocationHelper]::GetLatestSDKTargetPlatformVersion('Windows', '10.0'))</LatestTargetPlatformVersion>
+    <WindowsTargetPlatformVersion Condition="'$(LatestTargetPlatformVersion)' &lt;= '10.0.22000.0'">$(LatestTargetPlatformVersion)</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion Condition="'$(LatestTargetPlatformVersion)' &gt; '10.0.22000.0'">10.0.22000.0</WindowsTargetPlatformVersion>
```

Comment:
Shouldn't we use `&gt;=` here to include the "equals" case here as well?

Reply:
"Equals" case is included in the other one with `&lt;=` (`<=`).

Reply:
Oh wow I totally missed the `=` there. I need coffee 😋